ium.py

```
@@ -1110,30 +1151,58 @@ def _update_container(self, container_name: str, image: str, tag: str,
 
             # Stop the container
             self.logger.info(f"Stopping container {container_name}...")
```

Ruff (0.16.0)

[warning] Logging statement uses f-string

(G004)